src/main/java/com/serjltt/moshi/adapters/Wrapped.java

```diff
@@ -64,22 +64,37 @@
 @Documented
 @JsonQualifier
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER })
+@Target({ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER})
 public @interface Wrapped {
   /** The path to the wrapped json value. */
-  String[] value();
+  String[] path();
```

Comment:
yeah, the backwards compatibility is the only thing that stops me here... 
on the other hand `@Wrapped(value = { }, failOnNotFound = true)` looks ugly...
what are your thoughts?